pkg/controller/servicebinding/reconciler.go

```diff
@@ -116,15 +106,15 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		return DoneOnNotFound(err)
```

Comment:
For some reason, cannot comment on line above. Strange!

Line 105: logger.Error(err, "On retrieving service-binding-request instance.")

should remove `request` word

Reply:
done! [here](https://github.com/redhat-developer/service-binding-operator/pull/495/files#diff-7fe010c306d9dfe7af577ac9356c7a8bR105)